### PR TITLE
Added Row Tapped capability to bind a Command that gets executed if a row is tapped.

### PR DESCRIPTION
**Pull request by chrigi28**
We sometimes perform actions when a row is tapped, even when selection is disabled. To achieve this, I added a GestureRecognizer to the row and set up a bindable command.

### CODE REVIEW

**Comment by akgulebubekir** [comment]
This functionality can be acheved by listening the `SelectedItem` property. What is the main purpose for this functionality?

**Comment by chrigi28** [comment]
We disable the selection mode because we don’t want the highlight effect on the active row. Setting the active row color to transparent will change its appearance to match the background color of the grid itself. Unfortunately, I couldn’t find a way to retrieve the current color setting. If you know of a better approach, please  me know

thx

File: Maui.DataGrid/DataGrid.xaml.cs [hunk 1]
```diff
@@ -289,6 +289,12 @@ private void SortAndPaginate(SortData? sortData = null)
                     }
                 }
             });
+    
+    /// <summary>
+    /// Gets or sets the Row Tapped Command.
+    /// </summary>
+    public static readonly BindableProperty RowTappedCommandProperty =
+        BindableProperty.Create(nameof(RowTappedCommand), typeof(ICommand), typeof(DataGrid), null);
```
Comment (akgulebubekir) [inline Maui.DataGrid/DataGrid.xaml.cs:297]: You should use `BindablePropertyExtensions.Create<DataGrid, ICommand> ...` for consistency.

**Comment by akgulebubekir** [comment]
I see, it makes sense. Just have a small comment. Once fixed I'll merge.

**Comment by chrigi28** [comment]
Cool, Thank you!

File: Maui.DataGrid.Sample/ViewModels/MainViewModel.cs [hunk 2]
```diff
@@ -23,6 +24,7 @@ public MainViewModel()
         Commands.Add("CompleteEdit", new Command(CmdCompleteEdit));
         Commands.Add("Edit", new Command<Team>(CmdEdit));
         Commands.Add("Refresh", new Command(CmdRefresh));
+        Commands.Add("Tapped", new Command(Tapped));
```
Comment (akgulebubekir) [inline Maui.DataGrid.Sample/ViewModels/MainViewModel.cs:27]:
```suggestion
        Commands.Add("Tapped", new Command(CmdTapped));

```

File: Maui.DataGrid.Sample/ViewModels/MainViewModel.cs [hunk 3]
```diff
@@ -136,4 +138,12 @@ private async void CmdRefresh()
         await Task.Delay(3000);
         IsRefreshing = false;
     }
+
+    private void Tapped(object item)
```
Comment (akgulebubekir) [inline Maui.DataGrid.Sample/ViewModels/MainViewModel.cs:142]:
```suggestion
    private void CmdTapped(object item)

```

**Review by akgulebubekir — APPROVED** [review]

**Pull request merged**